src/transformers/models/wav2vec2_bert/feature_extraction_wav2vec2_bert.py

```diff
@@ -0,0 +1,302 @@
+# coding=utf-8
+# Copyright 2024 The HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Feature extractor class for Wav2Vec2-BERT
+"""
+
+from typing import List, Optional, Union
+
+import numpy as np
+
+from ...audio_utils import mel_filter_bank, spectrogram, window_function
+from ...feature_extraction_sequence_utils import SequenceFeatureExtractor
+from ...feature_extraction_utils import BatchFeature
+from ...utils import PaddingStrategy, TensorType, is_torch_available, logging
+
+
+if is_torch_available():
+    import torch
+
+logger = logging.get_logger(__name__)
+
+
+# Copied from transformers.models.seamless_m4t.feature_extraction_seamless_m4t.SeamlessM4TFeatureExtractor with input_features->input_values, SeamlessM4T->Wav2Vec2BERT
```

Comment:
Once we update `input_values` -> `input_features` in the modelling code, we can just use the `SeamlessM4TFeatureExtractor` directly. This FE file can then be removed in favour of using the existing class.

Let's try to then and advocate the use of the `AutoFeatureExtractor` API for loading the FE in the docs and examples, such that the user does not have to worry about loading the correct FE class: https://huggingface.slack.com/archives/C01N44FJDHT/p1667824904971239?thread_ts=1667816128.702279&cid=C01N44FJDHT